dev/docker-compose.yaml

```diff
@@ -10,10 +10,8 @@ x-airflow-common:
   environment:
     &airflow-common-env
     DB_BACKEND: postgres
-    AIRFLOW__CORE__EXECUTOR: CeleryExecutor
+    AIRFLOW__CORE__EXECUTOR: LocalExecutor
```

Comment:
I think time will tell us if `LocalExecutor` will limit the tests we run with `Tilt` or not - we can always revert this change if we face issues in the future.

Reply:
yeah that's what I'm thinking too - worth noting that I checked and the Astro CLI uses the `LocalExecutor`, so this is consistent with an end user's local dev experience